docs/specs/environment-specific-sms-gateway.md

```diff
@@ -0,0 +1,141 @@
+# Environment specific SMS Gateway
+
+This document describes configuration of sms gateway.
+
+- [Using environment specific sms gateway](#using-environment-specific-sms-gateway)
+  - [Configuration](#configuration)
+- [Using Custom SMS Gateway](#using-custom-sms-gateway)
+  - [Configuration](#custom-sms-gateway-configuration)
+  - [Webhook](#webhook)
+  - [Deno Hook](#deno-hook)
+
+## Using environment specific sms gateway
+
+### Configuration
+
+The following configs must be specified in `.env`:
```

Comment:
> The following environment variables are added

Rationale, they do not actually know to live in `.env`, they are just environment variables.